### DISCUSSION TITLE
I need a help

### DISCUSSION THREAD

**Discussion by PARKER-X** [GeriLife/companionship-care#214] [Q&A] [answered]
If you created a room for chatting for circle each circle has own chatting room. How can this is implemented 
How can i write models for chatt.  for example if a circle name has avenger so that circle has own chatting room. It is similar to our app aditional we also add a room for chatting.
I need a detail explanation how can i implement this. 
Thanks You!!

**Comment by brylie** [comment, marked as answer]
Try following this tutorial: https://justdjango.com/blog/chat-app-django-channels

It may also be useful to do a web search for "django chat tutorial" to find other examples.
